src/main/java/com/scottlogic/TopicSort.java

```diff
@@ -0,0 +1,67 @@
+package com.scottlogic;
+
+import com.scottlogic.sorts.KeywordSearch;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.*;
+
+public class TopicSort {
+    public TreeMap<String, List<UserPost>> topicMap = new TreeMap<>();
```

Comment:
I haven't come across a TreeMap before. Is it similar to a HashMap?

Reply:
A TreeMap will automatically sort based on the key :)